marshal.go

```diff
@@ -168,3 +168,7 @@ func WithDataSizeLen(l int) MarshalOption {
 		return nil
 	}
 }
+
+func LintError() {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported function LintError should have comment or be unexported